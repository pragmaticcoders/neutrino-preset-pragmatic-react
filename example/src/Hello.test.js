import React from 'react';
import Hello from './Hello';


describe('(Component) Hello', function() {

  it('should render', function() {
    const wrapper = shallow(
      <Hello name="World"/>
    );
    expect(wrapper).toMatchSnapshot();

  });

});
