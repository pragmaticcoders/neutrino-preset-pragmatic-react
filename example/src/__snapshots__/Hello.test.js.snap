// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Hello should render 1`] = `
<div
  className="Hello"
>
  <h1
    className="Hello"
  >
    Hello 
    World
    !
  </h1>
</div>
`;
